BITGROOM/config/cmake/binex/example/testfiles/h5ex_d_bitgroom.ddl

```diff
@@ -0,0 +1,209 @@
+HDF5 "h5ex_d_bitgroom.h5" {
+GROUP "/" {
+   DATASET "DS1" {
+      DATATYPE  H5T_STD_I32LE
+      DATASPACE  SIMPLE { ( 32, 64 ) / ( 32, 64 ) }
+      STORAGE_LAYOUT {
+         CHUNKED ( 4, 8 )
+         SIZE 8192 (1.000:1 COMPRESSION)
+      }
+      FILTERS {
+         USER_DEFINED_FILTER {
+            FILTER_ID 32008
+            COMMENT bitgroom; see https://github.com/kiyo-masui/bitgroom
```

Comment:
Is this correct for the BitGroom plugin?

Reply:
That is from the repo file

Reply:
Fixed